tests/Unit/Css/DocumentTest.php

```diff
@@ -33,7 +33,7 @@ final class CssDocumentTest extends TestCase
     public function parsesSelector(string $selector): void
     {
         $css = $selector . '{ color: green; }';
-        $subject = new CssDocument($css);
+        $subject = new Document($css);
```

Comment:
Though in the `TestCase` I think it's fine to `use` it `as` `Document`, as it's quite clear this is the class under test.